packages/oui-header-tabs/src/header-tab.html

```diff
@@ -0,0 +1,11 @@
+<li class="oui-tabs__tab"
+    data-ng-class="{ 'oui-tabs__tab_active': $ctrl.active, 'oui-tabs__tab_animate': $ctrl.isActivating }">
+    <a data-ng-if="$ctrl.state && !$ctrl.disabled"
+        aria-expanded="{{$ctrl.active ? 'true' : 'false' }}"
+        ui-sref="{{ $ctrl.state + '(' + String($ctrl.stateParams) + ')' }}">
+        {{$ctrl.text}}
+    </a>
+    <a data-ng-if="$ctrl.state && $ctrl.disabled" class="oui-tabs__tab_disabled">
```

Comment:
Should be `span` if it's a link without `href`.
The modifier class should be with `oui-tabs__tab` on the `li`. We don't use modifier class alone